baseframe/templates/baseframe/bootstrap3/forms.html.jinja2

```diff
@@ -111,6 +111,9 @@
   {% if message %}<p>{{ message }}</p>{% endif %}
   <div style="display:none;"><input type="hidden" name="form.id" value="{{ formid }}" /></div>
   {{ form.hidden_tag() }}
+  {%- if autosave %}
+    <input type="hidden" name="form.revision" id="revision" value="{{ last_revision_id if last_revision_id is not none }}" />
```

Comment:
It is unsafe to use `id="revision"` because there could be multiple forms on the page. This is where `formid` matters. It's safe to use `id="{{ formid }}-revision"`, or to skip the id entirely and use CSS selector `form#{{ formid }} > input[name="form.revision"]`